docs/src/filters/writing_custom_filters.md

```diff
@@ -392,14 +387,13 @@ However, it usually contains a Protobuf equivalent of the filter's static config
        }
        ```
 
-[Filter]: #
-[FilterFactory]: #
-[filter-factory-name]: #FilterFactory::name
-[FilterRegistry]: #
-[FilterChain]: #
-[runner]: #
-[create-filter-args-config]: #CreateFilter::config
-[config-type-dynamic]: #ConfigType::Dynamic
+[Filter]: https://docs.rs/quilkin/0.1.0/quilkin/filters/trait.Filter.html
```

Comment:
So that it always stays inline with the released version, we can link directly to our local versions of the api docs, but relatively.

```suggestion
[Filter]: ../../api/quilkin/filters/trait.Filter.html
```
(I think that's the right number of redirects, might want to confirm on the [`gh-pages` branch](https://github.com/googleforgames/quilkin/tree/gh-pages))

And repeat as per below.

Then we never have to update the version 🙂

Because of this, created #365 - to make it easier to preview this locally.

Reply:
do we have any means of versioning what we deploy with mdbook? e.g if we release a 0.2.0 and later a 0.3.0 as far as the mdbook doc is concerned 0.2.0 version no longer existed right? or is that kept around still?

Reply:
Yep! @XAMPPRocky did an awesome job there - the github action runs on each tag run ([source](https://github.com/googleforgames/quilkin/blob/main/.github/workflows/docs.yml)), and grabs the tag/branch name and makes it the parent folder (so development is in `main`, but when we make a v0.2.0 release we'll get a `v0.2.0` parent folder).

I tested it out on a fork, and it worked super nice 👍🏻

Reply:
Cool!